builder/bundles.go

```diff
@@ -535,7 +532,7 @@ func buildOsCore(b *Builder, packagerCmd []string, chrootDir, version string) er
 		return err
 	}
 
-	if err := fixOSRelease(b, filepath.Join(chrootDir, "usr/lib/os-release"), version); err != nil {
+	if err := fixOSRelease(b, filepath.Join(chrootDir, "usr/lib/os-release"), version, chrootDir); err != nil {
```

Comment:
Its not part of this change, but can we call this function `updateOSReleaseFile()`?
Lets also add a print statement inside the function, `fmt.Println("... updating os-release file")`